OneCommunityGlobal/HighestGoodNetworkApp
# Xiao F add floating and dismissable team notification header bar

# Description
![CleanShot 2023-11-22 at 16 28 21](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/f52c126b-0e68-4afe-9836-b0fb32d2592f)


## Related PRS (if any):
N/A

## Main changes explained:
- Update file `src/components/Dashboard/Dashboard.jsx` for including new header bar and its time logic
- Update various other files to implement styling of the header bar

## How to test:
1. Check into current branch
2. Do `npm install` and `npm run start:local` to run this PR locally
3. Log in as volunteer/manager on a team (Check if a person is on a team by going to its profile)
4. Go to dashboard -> Should see a blue header bar with different text according to the role
5. Dismiss the header and the header will not show up again until next Thursday
6. Log in as any other roles -> Should not see the blue header
7. Change the system time to next Friday
8. Log out and log in again as volunteer/manager ->Should see the blue header again

## Screenshots or videos of changes:

For volunteer on a team:
![CleanShot 2023-11-22 at 17 38 31](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/9832d165-1f83-41be-a0a1-760630102543)

For manager on a team:
![CleanShot 2023-11-22 at 17 38 59](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/c42ea36a-fb45-4ab0-a2ea-b25e1682e3df)


Test video:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/ddb4201c-0162-4fc5-98c0-e9e69399a00f

## Note:
N/A
